packages/ListBox/src/components/Option/Option.js

```diff
@@ -0,0 +1,80 @@
+import React from "react";
+import PropTypes from "prop-types";
+import useListBox from "../../useListBox";
+import { getA11yAttributesForOption } from "../../helpers/DOMAttributes";
+import { isOptionVisible, isOptionSelected, handleClickOption } from "../Options/helpers/options";
+import { OptionStyled } from "./Option.styles";
+
+const propTypes = {
+  /** String, number or JSX content */
+  children: PropTypes.oneOfType([PropTypes.node, PropTypes.func]).isRequired,
+
+  /** Describe if the option is selected or not */
+  isSelected: PropTypes.bool,
+
+  /** Describe if the option is enable or not */
+  isDisabled: PropTypes.bool,
+
+  /** Describe if the option is hidden or not */
+  isHidden: PropTypes.bool,
+
+  /** When the children are not a String, label should need to be add so the filter can work  */
+  label: PropTypes.string,
+
+  /** Callback for the clicking event */
+  onClick: PropTypes.func,
+
+  /** Value of your option this can be any data structure  */
+  value: PropTypes.any, // eslint-disable-line
+};
+
+const defaultProps = {
+  isDisabled: false,
+  isHidden: false,
+  // internal prop, which shouldn't be documented
+  preventDefaultOnSelect: false, // eslint-disable-line
```

Comment:
It's not in `propTypes`... can it actually be removed now?

Reply:
is not in propTypes because is an internal prop. this makes possible to have `<ListBox.RawItem />` which pretty much is an alias of `<ListBox.Option preventDefaultOnSelect />`.